fs.js

```diff
@@ -69,8 +103,32 @@ exports.makeDirSync = path => {
 	}
 };
 
+/**
+ * @param {import("fs").PathLike} source
+ * @param {import("fs").PathLike} destination
+ * @param {number?} [flags]
+ */
```

Comment:
I prefer the doc style without so many noisy star characters. Would be great if you could use this style everywhere. Also single-quotes.

```suggestion
/**
@param {import('fs').PathLike} source
@param {import('fs').PathLike} destination
@param {number?} [flags]
*/
```